Menuetti/wwwroot/js/site.js

```diff
@@ -4,14 +4,14 @@
 // Write your JavaScript code.
 
 /*setting the previous recipe id to local storage for later shopping list creation and hiding recipe details*/
-function fnSaveChoiceBack() {
-    localStorage.setItem('recipe0', 0)
-    $(".DetCarousel").hide();
-}
+//function fnSaveChoiceBack() {
```

Comment:
Is there any reason to leave these even as comments? Come on, if not in use, just remove pls :)